packages/cli/src/cmds/build/bundle.ts

```diff
@@ -32,7 +32,9 @@ export function bundle(
     if (transpilationMode !== TranspilationModes.none) {
       bundler.transform(require('babelify'), {
         global: transpilationMode === TranspilationModes.localAndDeps,
+        extensions: ['.js', '.ts'],
         presets: [
+          require('@babel/preset-typescript'),
```

Comment:
I didn't consider this solution. Nice :+1:

Reply:
We should note in the readme or usage instructions that transpilation needs to be at least local for TypeScript projects.